src/main/java/Event.java

```diff
@@ -0,0 +1,23 @@
+public class Event extends Deadline {
+    protected String deadlineTill;
```

Comment:
Perhaps it would be a good idea to name this something like "endTime" since events generally have a start and end time instead of using deadlineTill.